Gemfile.lock

```diff
@@ -1,47 +1,52 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (3.0.0)
+    CFPropertyList (3.0.2)
     activesupport (4.2.11.1)
       i18n (~> 0.7)
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     addressable (2.6.0)
       public_suffix (>= 2.0.2, < 4.0)
+    algoliasearch (1.27.1)
+      httpclient (~> 2.8, >= 2.8.3)
+      json (>= 1.5.1)
     atomos (0.1.3)
     babosa (1.0.2)
     claide (1.0.3)
-    cocoapods (1.6.1)
+    cocoapods (1.8.4)
```

Comment:
had to upgrade cocoapods, 1.6.1 was outputting errors. I think it doesn't support the new format of `simctl` in Xcode 10 or 11.